src/charactersheet/models/common/player_types.js

```diff
@@ -3,14 +3,14 @@ export var PlayerTypes = {
         key: 'character',
         visibleTabs: [
             'profile', 'stats', 'skills', 'spells', 'equipment',
-            'inventory', 'notes', 'settings', 'party', 'chat', 'exhibit'
+            'inventory', 'notes', 'settings', 'party', 'exhibit'
         ],
         defaultTab: 'stats'
     },
     dm: {
         key: 'dm',
         visibleTabs: [
-            'encounter', 'overview', 'dmscreen', 'party', 'chat', 'notes'
+            'encounter', 'overview', 'dmscreen', 'party', 'notes', 'exhibit'
```

Comment:
I added it to the DM and the reason is in my PR:

> DMs can now see their own exhibit, in case they want to know what the players will see (i.e. a preview).

I've also considered using this on my side. A DM could use a tablet or something to show off the exhibit from their own account.

Reply:
I'm +0 on it. A DM seeing their own exhibit is them just looking at the image they already have.